SDL/Triplet.h

```diff
@@ -941,6 +940,16 @@ namespace SDL {
                                                         zHiPointed,
                                                         sdlCut,
                                                         betaInCut);
+            if (!success) {
+              zOut = -999;
+              rtOut = -999;
```

Comment:
I don't think this is the right place to add this change.

`if (!success)` case is not going to be used at all.
The point was to add the initialization inside `runTripletConstraintsAndAlgo`

Did the compiler warning even go away about `betaIn`?

Reply:
I was looking at the underlying calls and for `betaIn`, there is no initialization of it in `passPointingConstraintBBB`

For all other variables, did the compiler complain about them being not initialized?

Reply:
No, only the betaIn complains. Other variables don't have complaints